"""Exceptions"""
class TestError(Exception):
    pass